lib/octokit/client/actions_secrets.rb

```diff
@@ -0,0 +1,53 @@
+module Octokit
+  class Client
+
+    # Methods for the Actions Secrets API
+    #
+    # @see https://developer.github.com/v3/actions/secrets/
+    module ActionsSecrets
+
+      # Get public key for secrets encryption
+      #
+      # @param repo [Integer, String, Hash, Repository] A GitHub repository
+      # @return [Hash] key_id and key
+      def get_public_key(repo)
+        get "#{Repository.path repo}/actions/secrets/public-key"
```

Comment:
A couple of really helpful things on all the methods would be adding the developer docs link and an options hash, so it would look something like this:
```suggestion
     # @see https://developer.github.com/v3/actions/secrets/#get-your-public-key
      def get_public_key(repo, options)
        get "#{Repository.path repo}/actions/secrets/public-key", options
```

Reply:
Created https://github.com/octokit/octokit.rb/pull/1236/commits/cdd9c749769bc7a997e0d12ffab055ae27174f14 based on this comment.